cli/azd/resources/scaffold/templates/resources.bicept

```diff
@@ -16,7 +16,7 @@ param {{.Name}} {{.Type}}
 param principalId string
 
 var abbrs = loadJsonContent('./abbreviations.json')
-var resourceToken = uniqueString(subscription().id, resourceGroup().id, location)
+var resourceToken = substring(uniqueString(subscription().id, resourceGroup().id, location), 0, 5)
```

Comment:
the original output of `uniqueString` func is 13 chars, here we get a substring to avoid possibly name length exceed.

Reply:
It's better to change the prefix instead of change the token. 
Which resource will exceed the length limit?

Reply:
> Which resource will exceed the length limit?

I got it, this one: `{{bicepName .Name}}_Postgres_${resourceToken}`

Reply:
Revert it first.